themes-default/slim/views/apiBuilder.mako

```diff
@@ -9,12 +9,6 @@
         <meta name="robots" content="noindex, nofollow">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <!-- These values come from css/dark.css and css/light.css -->
-        % if app.THEME_NAME == "dark":
-        <meta name="theme-color" content="#333333">
```

Comment:
I figured you were leaving one copy of this as I think this is still used for mobile apps when you pin the site to your homescreen.

Reply:
Oh I thought it wasn't used at all, didn't even occur to me to search Google for this meta tag.
I'll add one back.